docs/01-dynamic-type-tests.md

```diff
@@ -44,7 +44,7 @@ function bar(x: ?string): string {
 }
 {% endhighlight %}
 
-Read more about <a href="docs/nullable-types.html">Maybe Types</a>.
+Read more about <a href="/docs/nullable-types.html">Maybe Types</a>.
```

Comment:
let's do `../docs/nullable-types.html` so that it also works on facebook.github.io/flow/